### ISSUE TITLE
PASETO RFC

### ISSUE THREAD

**Issue by gcrow** [paseto-standard/paseto-rfc#5]
Are you still bringing this to IETF?

We need assurances that PASETO has life beyond paragonie.

**Comment by paragonie-security** [comment]
Yes, this will still be brought before the IETF for consideration. However, we anticipate getting sidelined by the JOSE working group and their members.

The alternative we're exploring is to publish PASETO as a C2SP specification. https://github.com/C2SP/C2SP

**Comment by gcrow** [comment]
Thanks!  Will watch both spaces.
